cmd/notary/integration_test.go

```diff
@@ -1289,6 +1289,177 @@ func TestClientKeyGenerationRotation(t *testing.T) {
 	}
 }
 
+// Tests key rotation
+func TestKeyRotation(t *testing.T) {
```

Comment:
👍 Thanks for adding these! 

Would you mind also adding one assertion that rotating to a key that doesn't exists fails?  :)